com.ca.lsp.cobol/lsp-core-cobol-parser/src/test/java/com/ca/lsp/core/cobol/AbstractCobolLinePreprocessorTest.java

```diff
@@ -24,8 +24,8 @@
 public abstract class AbstractCobolLinePreprocessorTest {
 
   protected ResultWithErrors<List<CobolLine>> processText(String text) {
-    CobolLineReaderImpl reader = new CobolLineReaderImpl(null);
-    return reader.processLines(text, FIXED);
+    CobolLineReaderImpl reader = new CobolLineReaderImpl();
+    return reader.processLines(null, text, FIXED);
```

Comment:
I think we should plan a small effort to have an method overloading here instead to pass null

Reply:
Yes, work in progress